### PR TITLE
[RG-137]: Play loading animation while compilation in tasks table view

### PR DESCRIPTION
**Pull request by TarasSydor**
> ### Motivate of the pull request
Given PR fulfills RG-137 issue and replaced Loading icon with an animation.

> ### Describe the technical details
Qt can automatically show icons in table view, but Gif animations aren't supported. To play it, I had to use QLabel as the widget for Status column, as set animation to it on each paint.
The class to play gif animation is QMovie.
>
> #### What does this pull request change?
Status column now shows loading animation instead of static icon while compiling the project:
image
image

> ### Which part of the code base require a change
> 
> - [ ] Library: <Specify the library name>
> - [ ] Plug-in: <Specify the plugin name>
> - [ ] Engine
> - [ ] Documentation
> - [ ] Regression tests
> - [ ] Continous Integration (CI) scripts

### CODE REVIEW

File: src/Compiler/TaskModel.cpp [hunk 1]
```diff
@@ -89,7 +89,7 @@ QVariant TaskModel::data(const QModelIndex &index, int role) const {
         case TaskStatus::Fail:
           return QIcon(":/failed.png");
         case TaskStatus::InProgress:
-          return QIcon(":/loading.png");
+          return QString(":/loading.gif");
```
Comment (KochynVolodymyr) [inline src/Compiler/TaskModel.cpp:92]: it doesn't make any sense. What is just return bool, e.g. true? or just 0 or 1 ?

Comment (ravic-rs) [reply src/Compiler/TaskModel.cpp:92]: @TarasSydor ?

Comment (TarasSydor) [reply src/Compiler/TaskModel.cpp]: fixed

**Review by KochynVolodymyr — APPROVED** [review]

**Review by ravic-rs — APPROVED** [review]
Looks good.

**Pull request merged**